info.md

```diff
@@ -36,6 +43,24 @@ Your vehicle must be in delivered status and in your posession for this integrat
 
 {% endif %}
 
+## Configuration
+
+To connect your newly installed integration to pull available sensor data (see the below example) for your Rivian vehicle, you should follow the following configuration recommendations. Not doing so could put your main Rivian account at risk of being locked and available.
+
+* One Time Password (**recommended**) - It is best security practice always to enable One Time Password / Multi-Factor Authentication. This way, your credentials (username + password) can not be used without your explicit consent through the personal device configured on your Rivian account.
```

Comment:
Until we have this working, we should leave this recommended step out

Reply:
It works now and closed the issue tracking it as well 😜